server/db/helper.js

```diff
@@ -67,9 +68,31 @@ const getCourse = (id) => {
     });
 };
 
+const getUserBadges = (userId) => {
+  return UserBadge.findAll({
+    where: {
+      id: userId,
```

Comment:
When searching for specific badges on the userBadge table you need to use the specific id and not the row's id
in this case it would be idUser instead of id